tests/input/inputfiletest.py

```diff
@@ -0,0 +1,26 @@
+import pypact as pp
+
+
+def test_reading_in_mass():
+    ff = pp.InputData()
+    pp.from_file(ff, 'examples/files/test.i')
```

Comment:
This could work, although I'd like to decouple the example files from the test files.

You can add the test file in the [`reference`](https://github.com/fispact/pypact/tree/b29640f6aebd5ad83e19c58997c284eb48e70aa5/reference) directory instead, that would then allow us to change the examples without impacting tests.